### PR TITLE
chore(build): docker multi-arch images

### PR DESCRIPTION
**Pull request by 24601**
add buildx config for linux/amd64 and linux/arm64 image build and push

### CODE REVIEW

File: distribution/pom.xml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (dadoonet) [inline distribution/pom.xml:3]: Could you revert this change?

File: distribution/pom.xml [hunk 2]
```diff
@@ -163,6 +166,7 @@
                             <external>
                                 <type>properties</type>
                                 <prefix>docker.ocr</prefix>
+
```
Comment (dadoonet) [inline distribution/pom.xml:169]: Could you revert this change?

**Review by dadoonet — CHANGES_REQUESTED** [review]
Thanks a lot for this.

Is there anything the end user has to do when he wants to pull the right image or is it done automatically by Docker based on the processor where Docker is running? (N00b question ;) )

File: distribution/pom.xml (hunk 1 above)
Comment (24601) [reply distribution/pom.xml]: Of course. Done!

File: distribution/pom.xml (hunk 2 above)
Comment (24601) [reply distribution/pom.xml]: Of course. Done!

**Comment by 24601** [comment]
> Thanks a lot for this.
> 
> Is there anything the end user has to do when he wants to pull the right image or is it done automatically by Docker based on the processor where Docker is running? (N00b question ;) )

No, the container runtime (and for sure Docker, but also podman, etc) tend to abstract this away from the user and require no special intervention for selecting platform images in most cases. The cases I have seen this does not happen are mostly edge cases in complicated/hybrid systems and architectures that are definitely edge cases.

**Comment by dadoonet** [comment]
Hey @24601 

Could you rebase your branch on master?

**Comment by Logicer16** [comment]
@24601 Would you mild quickly rebasing this so it can be merged

**Comment by dadoonet** [comment]
Changes made in #1729

**Pull request closed without merging**